[ffcall] Fails to build on ARMv8

I just picked up the Samsung Chromebook Plus tonight and noticed that `ffcall` fails to install properly.

```
chronos@localhost /usr/local/lib/crew/packages $ crew install ffcall
ffcall: Foreign function call is a collection of four libraries which can be used to build foreign function call interfaces in embedded interpreters.
http://www.haible.de/bruno/packages-ffcall-README.html
version 1.10-1
No precompiled binary available for your platform, downloading source...
######################################################################## 100.0%
Archive downloaded
Unpacking archive, this may take awhile...
gcc (GCC) 4.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
checking whether make sets $(MAKE)... yes
configure: creating ./config.status
config.status: creating Makefile
configure: configuring in avcall
configure: running /bin/bash './configure' --prefix=/usr/local  '--prefix=/usr/local' 'CFLAGS= -fPIC' --cache-file=/dev/null --srcdir=.
checking for gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ANSI C... none needed
checking how to run the C preprocessor... gcc -E
checking for egrep... grep -E
checking whether gcc needs -traditional... no
checking whether using GNU C... yes
checking for underscore in external names... no
checking for ranlib... ranlib
checking whether make sets $(MAKE)... yes
checking for a BSD compatible install... /usr/bin/install -c
checking how to copy files... cp -p
checking how to make hard links... ln
checking build system type... ./../autoconf/config.guess: line 1380: /tmp/cgLpHx5i/dummy: Permission denied
./../autoconf/config.guess: unable to guess system type

This script, last modified 2004-03-03, has failed to recognize
the operating system you are using. It is advised that you
download the most up to date version of the config scripts from

    ftp://ftp.gnu.org/pub/gnu/config/

If the version you run (./../autoconf/config.guess) is already up to date, please
send the following data and any information you think might be
pertinent to <config-patches@gnu.org> in order to provide the needed
information to handle your system.

config.guess timestamp = 2004-03-03

uname -m = aarch64
uname -r = 4.4.70-11386-g8271523ecf43
uname -s = Linux
uname -v = #1 SMP PREEMPT Sun Oct 15 17:38:26 PDT 2017

/usr/bin/uname -p = ARMv8 Processor rev 4 (v8l)
/bin/uname -X     = 

hostinfo               = 
/bin/universe          = 
/usr/bin/arch -k       = 
/bin/arch              = 
/usr/bin/oslevel       = 
/usr/convex/getsysinfo = 

UNAME_MACHINE = aarch64
UNAME_RELEASE = 4.4.70-11386-g8271523ecf43
UNAME_SYSTEM  = Linux
UNAME_VERSION = #1 SMP PREEMPT Sun Oct 15 17:38:26 PDT 2017
configure: error: cannot guess build type; you must specify one
configure: error: /bin/bash './configure' failed for avcall
ffcall failed to install: `./configure --prefix=/usr/local CFLAGS=" -fPIC"` exited with 1
```

--- a/packages/ffcall.rb
+++ b/packages/ffcall.rb
@@ -1,34 +1,23 @@
 require 'package'
 
 class Ffcall < Package
-  description 'Foreign function call is a collection of four libraries which can be used to build foreign function call interfaces in embedded interpreters.'
-  homepage 'http://www.haible.de/bruno/packages-ffcall-README.html'
-  version '1.10-1'
-  source_url 'http://www.haible.de/bruno/gnu/ffcall-1.10.tar.gz'
-  source_sha256 '6f1b5b8fc84b2c0051637fb1e4e4f8b975f5f98bff8fe053c1992347baa4983d'
+  description 'GNU libffcall is a library which can be used to build foreign function call interfaces in embedded interpreters.'
+  homepage 'https://www.gnu.org/software/libffcall/'
+  version '2.0'
+  source_url 'https://ftp.gnu.org/gnu/libffcall/libffcall-2.0.tar.gz'
+  source_sha256 '41aad2b449d90a5308e72267829db006dc9f9cc8736807082e1f2daef680196c'
 
   binary_url ({
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ffcall-1.10-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ffcall-1.10-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-       i686: '9d64fa08b420b2e0e04545d905fd3a06ee3c42c9fb6770d41f455395a4255344',
-     x86_64: 'd7a2d2cb69fa4ce1a9e9da530a2fa3228b48a9d4c70195f73b96eea9237b543e',
   })
 
   def self.build
-    system "./configure --prefix=/usr/local CFLAGS=\" -fPIC\""
-
-    # force to compile in sequential since ffcall Makefile doesn't work in parallel
-    system "make", "-j1"
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} CFLAGS=\" -fPIC\""
+    system 'make', '-j1' # doesn't support parallel builds
   end
 
   def self.install
-    system "make check"
-
-    # ffcall's `make install` doesn't create intermediate directory, so prepare for it here.
-    system "mkdir", "-p", "#{CREW_DEST_DIR}/usr/local"
-
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
 end

